README.md

```diff
@@ -2,7 +2,7 @@
 
 # Math Magicians
```

Comment:
- [ ] Let's try to be thorough in our documentation as it can be of great help to users. Your link for cloning the repo as contained in the README is incorrect. This is the correct link: `https://github.com/Gegardus/math-magicians.git`
- [ ] After installing the packages with `npm install`, kindly add the command for running the app ie `npm start`.

  ![gegardus-readme](https://user-images.githubusercontent.com/36542986/154202346-b408501a-2357-4f93-a693-241e3fd30e32.jpg)

Reply:
Changes made.